app/controllers/concerns/current_project.rb

```diff
@@ -0,0 +1,18 @@
+module CurrentProject
+  extend ActiveSupport::Concern
+
+  included do
+    helper_method :current_project
+  end
+
+  def current_project
+    @project
+  end
```

Comment:
As I replied above, I also like this approach. Went ahead with the refactoring that was needed.